src/components/fields/image-upload/image-review/image-review.styles.ts

```diff
@@ -6,10 +6,18 @@ import { Modal } from "@lifesg/react-design-system/modal";
 import { Text } from "@lifesg/react-design-system/text";
 import styled, { css } from "styled-components";
 
-export const ModalBox = styled(Modal.Box)<{ statusBarHeight: number }>`
+interface IModalBoxStyle {
+	imageReviewModalStyles?: string | undefined;
+}
+
+export const ModalBox = styled(Modal.Box)<IModalBoxStyle>`
 	display: block;
 	max-height: fit-content;
 
+	${({ imageReviewModalStyles }) => {
+		if (imageReviewModalStyles) return `${imageReviewModalStyles}`;
+	}}
```

Comment:
this will technically be a breaking change. can we do it in a non-breaking manner

Reply:
in Location it also using raw css when pass into FEE,
but it using mastheadHeight in web FE to build css in the schema

Reply:
ok